src/xdebugConnection.ts

```diff
@@ -679,11 +679,6 @@ export class Connection extends DbgpConnection {
         const data = iconv.encode(commandString, ENCODING)
         this._pendingCommands.set(transactionId, command)
         this._pendingExecuteCommand = command.isExecuteCommand
-        if (this._pendingExecuteCommand) {
```

Comment:
And this flag?

Reply:
@felixfbecker The flag itself is still used to prevent that deadlock where it waits for breakpoint init call on a connection that is hung running code.